server/data/visitSchedulerApiClient.test.ts

```diff
@@ -163,4 +163,36 @@ describe('visitSchedulerApiClient', () => {
       expect(output).toEqual(createSessionTemplateDto)
     })
   })
+
+  describe('getLocationGroups', () => {
```

Comment:
While I was at it, @hutcheonb-moj , I added a test here and a service level one for locations